ex11/part3/src/randomLabelCheck.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+
+export default function randomLabelCheck(prop) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [textb] = useState(Math.floor(Math.random() * 1000));
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const labaleTextMap = {
```

Comment:
The idea of mapping is a good one, however the functionality here is simple enough so that you don't actually need this. Don't try to solve future problems by adding extra code, solve current problems with the simplest and most straightforward code.